Makefile

```
@@ -57,13 +57,18 @@ push_translations: ## push source strings to Transifex for translation
 
 pull_translations:  ## pull translations from Transifex
 	git clean -fdX conf/locale
+ifeq ($(OPENEDX_ATLAS_PULL),)
 	i18n_tool transifex pull
 	i18n_tool extract
 	i18n_tool dummy
 	i18n_tool generate --verbose 1
 	git clean -fdX conf/locale/rtl
 	git clean -fdX conf/locale/eo
 	i18n_tool validate --verbose
+else
+	find conf/locale -mindepth 1 -maxdepth 1 -type d -exec rm -r {} \;
+	atlas pull $(OPENEDX_ATLAS_ARGS) translations/edx-platform/conf/locale:conf/locale
+endif
 	paver i18n_compilejs
```

Please add `validate` (verified po files) and `generate` (creates mo files) for both steps

```suggestion
	i18n_tool validate --verbose
	i18n_tool generate --verbose 1
	paver i18n_compilejs
```

---

@shadinaif I'm not sure about `i18n_tool generate` but we need to compile `po` --> `mo` one way or the other.

Let's try `python manage.py compilemessages`